pkgs/development/interpreters/starlark/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+buildGoModule rec {
+  pname = "starlark";
+  version = "unstable-2022-02-13";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "starlark-go";
+    rev = "c55a923347b1ca17cb609b1570187075c6bb911f";
+    sha256 = "sha256-Str7PKQqbI0SniRJDompwlshHmC299EZzqEb4pYsd+8=";
+  };
+
+  vendorSha256 = "sha256-lgL5o3MQfZekZ++BNESwV0LeoTxwEZfziQAe99zm4RY=";
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/google/starlark-go";
+    description = "Starlark in Go: the Starlark configuration language, implemented in Go";
```

Comment:
https://nixos.org/manual/nixpkgs/stable/#var-meta-description says the package name shouldn't be repeated for shortness but I don't see any other good option here.